### PR TITLE
added snackbar

### PR DESCRIPTION
**Pull request by anshikaa12**
snackbardoc

### CODE REVIEW

File: components/snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+.success-snack {
+    background-color: #4E9A51;
+}
+
+.error-snack {
+    background-color: #D84646;
+}
+
+.alert-snack {
+    background-color: #F68A1C;
```
Comment (nerdyspook) [inline components/snackbar/snackbar.css]: you can declare global variables for colors, so that you will be consistent with your UI

**Review by nerdyspook — COMMENTED** [review]
It looks good

File: components/snackbar/snackbar.css (hunk 1 above)
Comment (anshikaa12) [reply components/snackbar/snackbar.css]: sure will do that!

**Review by nerdyspook — APPROVED** [review]

**Review by nikhilmalviya-dev — APPROVED** [review]
good work

**Pull request merged**